src/lib/lore.sh

```diff
@@ -415,7 +415,8 @@ function process_patchsets()
   local pids
   local i
 
-  shared_dir_for_parallelism=$(mktemp --directory)
+  shared_dir_for_parallelism=$(mktemp -p /dev/shm --directory)
```

Comment:
Before creating a folder inside `/dev/shm`, ensure it exists and that you have permission to change it. Also, check: https://kworkflow.org/content/codingstyle.html#avoid-short-options

Reply:
Agree, you should try using `/dev/shm` and fallback to the default directory of `mktemp --directory`.

Also, what you guys think of creating a dedicated function like `create_shared_memory_dir`, to add automated tests? Maybe inside `kwlib.sh`.